a10_octavia/common/utils.py

```diff
@@ -28,7 +28,18 @@
 
 def validate_ipv4(address):
     """Validates for IP4 address format"""
-    return netaddr.valid_ipv4(address, netaddr.core.INET_PTON)
+    if not netaddr.valid_ipv4(address, netaddr.core.INET_PTON):
```

Comment:
these changes are taken from feature branch - **feature/hierarchical_multitenancy** ie.
the refactored `utils.py` also having `partition_name` instead of `partition`